index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <H1>Hello Microverse</H1>
```

Comment:
- [ ] Good effort working on your first project as a student in microverse, I must really commend your effort. But I would kindly suggest you make your HTML tags be written in lowercase so as to follow the best practice of coding. As this tags can be case sensitive.

- [ ] Also i would advice you give the `h1` tag a class name that can be used to style the element in your CSS file.

E.g :point_down: 
```HTML
<h1 class="header">Hello Microverse</h1>
```